pyproject.toml

```diff
@@ -28,6 +28,12 @@ classifiers = [
 dynamic = ["version"]
 dependencies = [
   "bioio-base>=0.2.0",
+  "dask[array]>=2021.4.1",
+  "fsspec>=2022.8.0",
+  "numpy>=1.16,<2",
+  "PyYAML>=6.0",
+  "readlif>=0.6.4",
+  "xarray>=0.16.1,<2023.02.0",
```

Comment:
I think this maximum was set to avoid issues with Python3.8, check out `bioio` or `bioio-sldy` for examples of versions that support Python 3.9+

Reply:
resolved in [c2d764a](https://github.com/bioio-devs/bioio-lif/pull/3/commits/c2d764abf04c346e342f56f798a1f32b5f20539c)